Sources/MockoloFramework/Models/ClassModel.swift

```diff
@@ -55,7 +55,32 @@ final class ClassModel: Model {
         self.accessLevel = acl
     }
     
-    func render(with identifier: String, encloser: String, useTemplateFunc: Bool, useMockObservable: Bool, allowSetCallCount: Bool = false, mockFinal: Bool = false, enableFuncArgsHistory: Bool = false, disableCombineDefaultValues: Bool = false) -> String? {
-        return applyClassTemplate(name: name, identifier: self.identifier, accessLevel: accessLevel, attribute: attribute, declType: declType, metadata: metadata, useTemplateFunc: useTemplateFunc, useMockObservable: useMockObservable, allowSetCallCount: allowSetCallCount, mockFinal: mockFinal, enableFuncArgsHistory: enableFuncArgsHistory, disableCombineDefaultValues: disableCombineDefaultValues, initParamCandidates: initParamCandidates, declaredInits: declaredInits, entities: entities)
+    func render(
+        with identifier: String,
+        encloser: String,
+        useTemplateFunc: Bool,
+        useMockObservable: Bool,
+        allowSetCallCount: Bool = false,
+        mockFinal: Bool = false,
+        enableFuncArgsHistory: Bool = false,
+        disableCombineDefaultValues: Bool = false
+    ) -> String? {
+        return applyClassTemplate(
+            name: name,
+            identifier: self.identifier,
+            accessLevel: accessLevel,
+            attribute: attribute,
+            declType: declType,
+            metadata: metadata,
+            useTemplateFunc: useTemplateFunc,
+            useMockObservable: useMockObservable,
+            allowSetCallCount: allowSetCallCount,
+            mockFinal: mockFinal,
+            enableFuncArgsHistory: enableFuncArgsHistory,
+            disableCombineDefaultValues: disableCombineDefaultValues,
+            initParamCandidates: initParamCandidates,
+            declaredInits: declaredInits,
+            entities: entities
+        )
```

Comment:
I would like to make program more readable. if there is a better format, please let me know. :)